pynest/examples/eprop_plasticity/online_eprop_supervised_regression_neuromorphic_mnist.py

```diff
@@ -169,8 +168,11 @@
 # ~~~~~~~~~~~~~~
 # We proceed by creating a certain number of input, recurrent, and readout neurons and setting their parameters.
 # Additionally, we already create an input spike generator and an output target rate generator, which we will
-# configure later. Within the recurrent network, alongside a population of regular neurons, we introduce a
-# population of adaptive neurons, to enhance the network's memory retention.
+# configure later. Each input sample, featuring two channels, is mapped out to a 34x34 pixel grid. We allocate
+# Poisson generators to each pixel of the input image to simulate spike events. However, due to the observation
+# that some pixels either never record events or do so infrequently, we maintain a blacklist of these inactive
+# pixels. By omitting Poisson generators for pixels on this blacklist, we effectively reduce the total number of
+# input neurons and Poisson generators required, optimizing the network's resource usage.
```

Comment:
```suggestion
# configure later. Each input sample, featuring two channels, is mapped out to a 34x34 pixel grid. We allocate
# Poisson generators to each input image pixel to simulate spike events. However, due to the observation
# that some pixels either never record events or do so infrequently, we maintain a blocklist of these inactive
# pixels. By omitting Poisson generators for pixels on this blocklist, we effectively reduce the total number of
# input neurons and Poisson generators required, optimizing the network's resource usage.
```

Reply:
The word "blacklist" has for some people negative connotations. I don't think, we lose anything by renaming it to blocklist or denylist (or a similar synonym) which are also used in the context of computing. Variables and the file including `blacklist` would also need to be changed.